### PR TITLE
performance creator

### PR DESCRIPTION
**Pull request by yuvalk**
- remove pod after usage is displayed
- fix typo in performance-profile-creator execution

Version(s):

Issue:

Link to docs preview:

Additional information:

### CODE REVIEW

File: modules/cnf-running-the-performance-creator-profile.adoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ Password: ************
 +
 [source,terminal]
 ----
-$ podman run --entrypoint performance-profile-creator registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 -h
+$ podman run --rm --entrypoint performance-profile-creator registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 -h
```
Comment (fromanirh) [inline modules/cnf-running-the-performance-creator-profile.adoc:52]: nice idea, I like it

File: modules/cnf-running-the-performance-creator-profile.adoc [hunk 2]
```diff
@@ -90,7 +90,7 @@ Using this information you can set appropriate values for some of the arguments
 +
 [source,terminal]
 ----
-$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 --info log --must-gather-dir-path /must-gather
+$ podman run --entrypoint performance-profile-creator -v `pwd`/must-gather:/must-gather:z registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 --info log --must-gather-dir-path /must-gather
```
Comment (fromanirh) [inline modules/cnf-running-the-performance-creator-profile.adoc:93]: +1 for the general idea. I think `$( pwd )` is more readable than backticks (``) but I'll defer the final call to doc reviewers

**Review by ffromani — COMMENTED** [review]
LGTM for the command line changes

**Comment by yuvalk** [comment]
/close
this is covered by https://github.com/openshift/openshift-docs/pull/49527

**Pull request closed without merging**